docs/docker.rst

```diff
@@ -0,0 +1,160 @@
+..
+    This file is part of INSPIRE.
+    Copyright (C) 2017 CERN.
+
+    INSPIRE is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    INSPIRE is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+
+    In applying this licence, CERN does not waive the privileges and immunities
+    granted to it by virtue of its status as an Intergovernmental Organization
+    or submit itself to any jurisdiction.
+
+
+Docker (Linux)
+==============
+
+Docker is an application that makes it simple and easy to run processes in a container,
+which are like virtual machines, more resource-friendly. For a detailed introduction to the
+different components of a Docker container.
+
+Useful `Tutorial
+<https://www.digitalocean.com/community/tutorials/the-docker-ecosystem-an-introduction-to-common-components>`_.
+
+
+Inspire and Docker
+##################
+
+Get the latest Docker setup.
+
+.. note:: If you are using Mac, please build a simple box with ``docker-engine`` above ``1.10`` and
+         ``docker-compose`` above ``1.6.0``.
+
+Make sure you can run docker without ``sudo``.
+
+- ``id $USER``
+
+  If you are not in the ``docker`` group, run the following command and then restart ``docker``. If this doesn't work, just restart your machine :)
```

Comment:
That's because you need to refresh the groups your current user is in. That can be done by the newgrp command under, or re-logging in (with sudo, su, or well, restarting).